pkgs/development/libraries/science/math/sympow/default.nix

```
@@ -61,6 +61,11 @@ stdenv.mkDerivation rec {
   doInstallCheck = true;
   installCheckPhase = ''
     export HOME=$TMPDIR
+    "$out/bin/sympow" -curve "[1,2,3,4,5]" -moddeg | grep 'Modular Degree is 464'
+    echo "[1,-1,0,-79,289]" | "$out/bin/sympow" -analrank | grep ^"Analytic Rank is 4"
+    "$out/bin/sympow" -curve "[1,-1,0,-79,289]" -analrank | grep ^"Analytic Rank is 4"
+    "$out/bin/sympow" -curve "[0,1,1,-2,0]" -analrank | grep ^"Analytic Rank is 2"
+  '' + lib.optionalString (!stdenv.isAarch64) ''
     "$out/bin/sympow" -sp 2p16 -curve "[1,2,3,4,5]" | grep '8.3705'
   '';
```

```suggestion
  installCheckPhase =
    ''
      export HOME=$TMPDIR
      "$out/bin/sympow" -curve "[1,2,3,4,5]" -moddeg | grep 'Modular Degree is 464'
      echo "[1,-1,0,-79,289]" | "$out/bin/sympow" -analrank | grep ^"Analytic Rank is 4"
      "$out/bin/sympow" -curve "[1,-1,0,-79,289]" -analrank | grep ^"Analytic Rank is 4"
      "$out/bin/sympow" -curve "[0,1,1,-2,0]" -analrank | grep ^"Analytic Rank is 2"
    ''
    + lib.optionalString (!stdenv.isAarch64) ''
      "$out/bin/sympow" -sp 2p16 -curve "[1,2,3,4,5]" | grep '8.3705'
    '';
```
to appease our nixfmt overlords.